lib/plugins.js

```diff
@@ -365,12 +439,12 @@ function plugins (settings, deviceId) {
 
         const rate = rawRate.div(cashInCommission)
 
-        const lowBalanceMargin = new BN(1.05)
+        const lowBalanceMargin = new BN(0.95)
```

Comment:
Any specific reason for this change?

Reply:
Yeah, taking 5% off is multiplying by 0.95, not dividing by 1.05.